pypeit/wavemodel.py

```diff
@@ -717,7 +717,7 @@ def create_linelist(wavelength, spec, fwhm, sigdetec=2.,
 
 def create_OHlinelist(resolution, waveminmax=(0.8,2.6), dlam=40.0, flgd=True, nirsky_outfile=None,
                       fwhm=None, sigdetec=3., line_name='OH', file_root_name=None, iraf_frmt=False, 
-                      debug=False):
+                      vacuum=False, debug=False):
     """Create a synthetic sky spectrum at a given resolution, extract significant lines, and
     store them in a PypeIt compatibile file. The skymodel is built from nearIR_modelsky and
     includes black body at 250K, OH lines, and H2O lines (but only at lambda>2.3microns).
```

Comment:
I'm lost as to why vacuum is False here. All pypeit wavelengths and linelists are always in vacuum? Can you please explain?

Reply:
@jhennawi this is related to what I talked to you about a week ago when I was making the OH linelists. When I first run the `create_OHlinelist` script, the wavelengths of the OH lines were consistently different (~2Ang) than the ones that we have in other linelists (e.g, NIRES). Then, I figured out that the sky lines from Rousselot2000 should be already in vacuum, and because the script `create_OHlinelist` was converting those into vacuum by default, the wavelengths of the OH lines were double corrected. This is why I introduced the `vacuum`  parameter and set it to `False`.